presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -128,6 +175,19 @@ public boolean isFinished()
     @Override
     public ListenableFuture<?> isBlocked()
     {
+        if (!spillInProgress.isDone()) {
```

Comment:
Add a comment so it does not look like a bug to someone looking at the code later